guide.qmd

```diff
@@ -0,0 +1,178 @@
+---
+title: "openstatsguide"
+subtitle: "version 0.1"
+date: "10 Mar 2024"
+---
+
+We encourage developers of statistical software packages to follow this minimum set of good practices. 
+They can be easily remembered with the mnemonic bridge sentence:
```

Comment:
```suggestion
They can be easily remembered with the mnemonic:
```

"mnemonic" sounds more fluent than "mnemonic bridge sentence" (although I don't doubt that the latter is the official term).

Reply:
Quarto Document does not render. What libraries do I need?

ERROR: Unknown format dashboard

Error: Unknown format dashboard
    at resolveFormats (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:83181:19)
    at async renderContexts (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:82635:21)
    at async Object.renderFormats (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:82686:26)
    at async inputTargetIndex (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:83502:21)
    at async resolveInputTarget (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:83581:19)
    at async resolveItem (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:93624:26)
    at async navigationItem (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:93595:16)
    at async navbarEjsData (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:93514:28)
    at async initWebsiteNavigation (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:92809:29)
    at async Object.preRender (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:97739:9)

Reply:
You need Quarto 1.4